hrrecruitment/tests/phpunit/CRM/HRRecruitment/BAO/HRVacancyTest.php

```diff
@@ -49,8 +37,11 @@ protected static function _populateDB($perClass = FALSE, &$object = NULL) {
    */
   function testCreateGet() {
 
-    $permissionIndividual1Id = Contact::createIndividual();
-    $permissionIndividual2Id = Contact::createIndividual();
+    $params1 = array('first_name' => 'micky', 'last_name' => 'mouse');
+    $permissionIndividual1Id = $this->createContact($params1);
```

Comment:
> This createContact() method will end up duplicated in another extension tests

when we agree on a common way to share  common test methods between extensions then that's will not be a problem any more.  (we should discuss this on chat tomorrow by the way)
 

> The Contact::createIndividual() is a little bit faster than createContact(), because it doesn't use the API

I am not going to worry about some milliseconds here , let us solve the first problem and then we will not worry about this .